acceptance/bundle/exec/databricks-cli/profile-is-passed/from_bundle/script

```diff
@@ -0,0 +1,10 @@
+# Replace placeholder with an actual host URL and token
+envsubst < .databrickscfg > out && mv out .databrickscfg
+export DATABRICKS_CONFIG_FILE=.databrickscfg
+
+# Credentials will be picked up from .databrickscfg. Unset existing credentials.
+unset DATABRICKS_HOST
+unset DATABRICKS_TOKEN
+
+# This should use the profile configured in the bundle, i.e. PAT auth
+trace $CLI bundle exec -- $CLI current-user me
```

Comment:
Should not we run this twice, once with 'exec' and another time without, to see the difference? Otherwise how does the test show impact of 'exec' here?

Reply:
Why? The point of this test is to ensure `bundle run` does pass the profile information, which this test already successfully asserts.

Reply:
well, if exec does nothing with env and `$CLI current-user me` works without it, then we don't know if `exec` had effect on execution.

Reply:
Yeah fair point. In this case we can't differentiate. Removed the test. The other test case `acceptance/bundle/run/inline-script/databricks-cli/profile-is-passed/from_flag` does successfully differentiate though because we can see from the recorded requests that oauth was used.